example_test.go

```diff
@@ -15,21 +15,23 @@
 package crypto_test
 
 import (
-	"fmt"
+//"fmt"
```

Comment:
why comment all this out ?

Reply:
not used

Reply:
ah, i was getting `Example_Sha256 has malformed example suffix:` the fix was to remove the underscore